com.vrcfury.vrcfury/Editor/VF/Feature/BakeHapticSocketsBuilder.cs

```diff
@@ -233,6 +248,29 @@ public class BakeHapticSocketsBuilder : FeatureBuilder {
                             socket.unitsInMeters
                         );
                     }
+
+                    if (socket.IsValidPlugLength) {
+                        // Calculate `Plug Length` using `TPS_Pen_Penetrating - TPS_Pen_Root + 0.01` 
+                        var lastPlugLength = fx.NewFloat("LastPlugLength");
+                        var plugLength = math.Subtract(penTip, penRoot);
+                        directTree.Add(math.MakeSetter(plugLength, 0.01f));

```

Comment:
Why is this here?

Reply:
I think it was probably a mistake that the sender was ever 0.01, rather than some smaller value. The socket senders are already 0.001. Would it be acceptable to get rid of this and just reduce the sender radius for new plugs? It wouldn't be perfectly backward compatible, but at least new senders will do it right, and old senders will only be slightly off.

Reply:
Decided against making that change. I guess we'll live with the 0.01 being part of the spec from now on :P